lms/djangoapps/bookmarks/serializers.py

```diff
@@ -0,0 +1,44 @@
+"""
+Serializers for Bookmarks.
+"""
+from rest_framework import serializers
+
+from .models import Bookmark
+
+
+class BookmarkSerializer(serializers.ModelSerializer):
+    """
+    Class that serializes the Bookmark model.
+    """
+    def __init__(self, *args, **kwargs):
+        # Don't pass the 'fields' arg up to the superclass
+        try:
+            fields = kwargs['context'].pop('fields', [])
```

Comment:
Assuming you clean up the import issue, you can just put DEFAULT_FIELDS as the default value, correct?